notebooks/systemds_dev.ipynb

```diff
@@ -0,0 +1,882 @@
+            "20/07/18 05:06:21 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable\n",
+            "log4j:WARN No appenders could be found for logger (org.apache.sysds.api.DMLScript).\n",
+            "log4j:WARN Please initialize the log4j system properly.\n",
+            "log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.\n",
+            "ANTLR Tool version 4.5.3 used for code generation does not match the current runtime version 4.7ANTLR Runtime version 4.5.3 used for parser compilation does not match the current runtime version 4.7ANTLR Tool version 4.5.3 used for code generation does not match the current runtime version 4.7ANTLR Runtime version 4.5.3 used for parser compilation does not match the current runtime version 4.7Calling the Direct Solver...\n",
+            "Computing the statistics...\n",
+            "AVG_TOT_Y, 2.306291350486548\n",
+            "STDEV_TOT_Y, 0.41527871477713496\n",
+            "AVG_RES_Y, 5.730468777276343E-9\n",
+            "STDEV_RES_Y, 5.849014866902826E-8\n",
+            "DISPERSION, 3.144664287007204E-15\n",
+            "R2, 0.9999999999999905\n",
+            "ADJUSTED_R2, 0.9999999999999818\n",
+            "R2_NOBIAS, 0.9999999999999906\n",
+            "ADJUSTED_R2_NOBIAS, 0.9999999999999801\n",
+            "R2_VS_0, 0.9999999999999997\n",
+            "ADJUSTED_R2_VS_0, 0.9999999999999994\n",
+            "SystemDS Statistics:\n",
+            "Total execution time:\t\t0.102 sec.\n",
+            "Number of executed Spark inst:\t0.\n",
+            "\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "SUGac5w9ZRBQ",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Working with SystemDS **Standalone**\n",
+        "\n",
+        "(NOTE: Pay attention to *directories* and *relative paths*. :))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "g5Nk2Bb4UU2O",
+        "colab_type": "text"
+      },
+      "source": [
+        "##### 1. Set SystemDS environement variables\n",
+        "\n",
+        "These are useful for the `./bin/systemds` script."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "2ZnSzkq8UT32",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!export SYSTEMDS_ROOT=$(pwd)\n",
+        "!export PATH=$SYSTEMDS_ROOT/bin:$PATH"
+      ],
+      "execution_count": 15,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Tcxkh8cdUy1V",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!echo 'export SYSTEMDS_ROOT='$(pwd) >> ~/.bashrc\n",
+        "!echo 'export PATH=$SYSTEMDS_ROOT/bin:$PATH' >> ~/.bashrc"
+      ],
+      "execution_count": 16,
```

Comment:
It seems like after some debugging from my side that all the exports doesn't work inside this notebook. This also includes the echo to ~/.bashrc.
also fixing block `16` adding a `!source ~/.bashrc` does not work.

Reply:
I did `!cat ~/.bashrc`, could see `SYSTEMDS_ROOT`.

Reply:
so the point here is not that it does not put the lines into .bashrc, the point is that it does not use them for the following commands, if they did you would be able to change the `./bin/systemds` to `systemds`.
i did not have success with making this work with either of the two methods.